app/lib/common/widgets/like_button.dart

```diff
@@ -125,6 +125,7 @@ class _LikeButtonState extends State<LikeButton> with TickerProviderStateMixin {
                 children: [
                   InkWell(
                     onTap: () {
+                      debugPrint('like click --------------------------------');
                       if (widget.onTap != null) {
                         widget.onTap!();
                       }
```

Comment:
```suggestion
                    onTap: widget.onTap
```

Is there ever a realistic chance we have no onTap at all? Maybe we should make that field required, too...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/2e8359d97d1a136895ad8ed60567bf368815bc10